chain/default_store.go

```diff
@@ -150,6 +156,7 @@ func (store *DefaultStore) Load(ctx context.Context) error {
 		return errors.Errorf("expected genesis cid: %s, loaded genesis cid: %s", store.genesis, loadCid)
 	}
 
+	logStore.Infof("complete chain load, setting head to: %s", headTs.String())
```

Comment:
"Finished loading %d tipsets from %s".

Is `headTs.String()` the same as `tipCids` logged at the start? Better to make them the same format.

Reply:
Yes they have the same value